pkg/virt-controller/watch/testing/testing.go

```diff
@@ -101,3 +104,28 @@ func DefaultVirtualMachineWithNames(started bool, vmName string, vmiName string)
 func DefaultVirtualMachine(started bool) (*v1.VirtualMachine, *v1.VirtualMachineInstance) {
 	return DefaultVirtualMachineWithNames(started, "testvmi", "testvmi")
 }
+
+type interf interface {
+	Execute() bool
+}
+
+func deepCopyList(objects []interface{}) []interface{} {
+	for i := range objects {
+		objects[i] = objects[i].(runtime.Object).DeepCopyObject()
+	}
+	return objects
+}
+
+func SanityExecute(c interf, stores []cache.Store, g gomega.Gomega) {
```

Comment:
I think this is generic enough to be placed under `/pkg/testutils`.
In this way it can be also used by the virt-handler instead of duplicate things. WDYT?

Reply:
Makes sense to take this out of virt-controller specific path

Reply:
Done